osu.Game/Overlays/Rankings/Tables/RankingsTable.cs

```diff
@@ -68,13 +70,19 @@ private void load()
 
         protected abstract Drawable[] CreateAdditionalContent(TModel item);
 
-        protected override Drawable CreateHeader(int index, TableColumn column) => new HeaderText(column?.Header ?? string.Empty, HighlightedColumn());
+        protected virtual string HighlightedColumn => @"Performance";
```

Comment:
Can't have this be a string. How about index instead?

Reply:
FWIW that was there before (check a few lines down, it was a method before). I'd have to check but index might not be feasible since these tables are inheritance hell and like two or three tables used the same base class for some reason, and I think index of this column might be different between them. Ideally I'd say these tables should be separated entirely (even if by cost of some code duplication) but that is not immediately trivial.

Reply:
If that's the case I'd like to see that happen first, since that would also probably change how this PR looks. As it is I'm concerned localisation will immediately break this, also it seems a little fragile w.r.t. capitalisation, spelling, etc.

Reply:
I agree that this mechanism isn't the best (same with `GradeColumns`), but I couldn't really come up with a solution to the inheritance problem that'd be any kind of acceptable so far.

Right now some changes seem to be planned for the rankings section ([see here](https://github.com/ppy/osu-web/pull/5889#issuecomment-614512400)), so I think this whole PR has to wait for them anyway. I'll try to work on these goddamn tables in the meantime.

Reply:
This is still an issue, but I'm going to look past it and get this change merged as it is generally improving the code quality around this area (which was already pretty shocking).

Note that this isn't adding a string but just moving it up from lower in the class, where it still exists on `master`.